dashboard/.eslintrc.json

```diff
@@ -96,6 +96,7 @@
         "forbid": [">", "}"] // avoid warnings with single/double quotes
       }
     ],
+    "react/no-unknown-property": ["error", { "ignore": ["cds-layout"] }], // allow 'cds-layout' attribute
```

Comment:
After upgrading the deps for a previous PR, my local env started complaining about this `<div cds-layout ...>`  tag, so adding this eslint rule to ignore it (as it is required in Clarity).

Reply:
In fact, the PR failed because of the lack of this fix 😅  https://app.circleci.com/pipelines/github/antgamdia/kubeapps/2768/workflows/618c2248-ac94-4a19-bd3a-65323e4feb21/jobs/22024